### PR TITLE
Design branch

### PR DESCRIPTION
**Pull request by Phenomenalhub**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -14,10 +14,15 @@
...
         app:layout_constraintTop_toTopOf="parent" />
+
     <com.google.android.material.bottomnavigation.BottomNavigationView
         android:id="@+id/bottom_navigation"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_width="413dp"
+        android:layout_height="68dp"
+        android:background="@color/white"
+        app:itemRippleColor="#8C9DDF"
```
Comment (mhammerly) [inline app/src/main/res/layout/activity_main.xml:23]: an idea for polishing in later weeks: replace hardcoded colors with themes/styles. themes allow you to define all your colors and whatnot in one place which makes it easy to, for example, create a "light mode" theme and a "dark mode" theme

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_200">#8C9DDF</color>
+    <color name="purple_500">#8C9DDF</color>
     <color name="purple_700">#FF3700B3</color>
```
Comment (mhammerly) [inline app/src/main/res/values/colors.xml:5]: what are 200, 500, and 700? 200 and 500 look like they're now the same color, and they're pretty far from FF3700B3 which looks more pink to me

**Review by mhammerly — APPROVED** [review]
code looks good! for design-focused changes like this, it's usually a good idea to include some before/after screenshots of different parts of the app in the PR description. if that's a chore to go back and do this time i'll settle for seeing it in person tomorrow!

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (Phenomenalhub) [reply app/src/main/res/values/colors.xml:5]: Included my app color preference to remove the default color. Will work on this.

**Pull request merged**